controllers/update_pods.go

```diff
@@ -87,13 +87,24 @@ func (updatePods) reconcile(ctx context.Context, r *FoundationDBClusterReconcile
 func getPodsToUpdate(logger logr.Logger, reconciler *FoundationDBClusterReconciler, cluster *fdbv1beta2.FoundationDBCluster, podMap map[fdbv1beta2.ProcessGroupID]*corev1.Pod) (map[string][]*corev1.Pod, error) {
 	updates := make(map[string][]*corev1.Pod)
 
+	var unavailablePods int
+
 	for _, processGroup := range cluster.Status.ProcessGroups {
 		if processGroup.IsMarkedForRemoval() {
 			logger.V(1).Info("Ignore removed Pod",
 				"processGroupID", processGroup.ProcessGroupID)
 			continue
 		}
 
+		if cluster.Spec.MaxUnavailablePods.IntValue() > 0 {
```

Comment:
I have created a dedicated validation since SkipProcessGroups uses the ignore pending time that defaults to 5min. 
And I think it is simpler to check it separately than having to set two values to get the functionality. Specially because the default ignore pending time value results in Pending Pods not being counted unless they have been pending for more than 5 min.